### ISSUE TITLE
Step 1) in build instructions does not work on mac

### ISSUE THREAD

**Issue by bsin9h** [launchdarkly/hello-dotnet#7]
This command: `nuget Install-Package LaunchDarkly.ServerSdk` does not work on a Mac. 
Errors with: `Unknown command: 'Install-Package'`

Is the command Windows specific?

However, the following command does work, at least on a Mac:

`nuget install LaunchDarkly.ServerSdk`

**Comment by eli-darkly** [comment]
First, there's no single set of commands that is for Windows or Mac - the way you install a NuGet package just depends on what tools you are using. For instance, if you're using Visual Studio, you're more likely to install packages through the menu commands for that, rather than using the command line - although Visual Studio also has its own extra command-line interface, the NuGet Package Manager Console, and I believe that the command you're looking at there is for that.

Basically, installing NuGet packages is such a standard thing to do in .NET development that it wasn't our intention to explain all the ways it can be done. The point of that example was just to show that `LaunchDarkly.ServerSdk` is the name of the package.

**Comment by eli-darkly** [comment]
Also, did you file this issue in the wrong repository? This one, `hello-dotnet`, is an example application that already has the SDK package installed. Its [build instructions](https://github.com/launchdarkly/hello-dotnet/blob/master/README.md) don't include any such command.

**Comment by alohaninja** [comment]
@eli-darkly - should be able to close this issue - I successfully used `dotnet add package Launchdarkly.ServerSdk` fine a Mac - just needed to have `dotnet-sdk` installed as a prerequisite.
